portal/account/register.php

```diff
@@ -0,0 +1,556 @@
+                            <div class="form-group inline">
+                                <label class="control-label" for=""><?php echo xlt('Group Number')?></label>
+                                <div class="controls inline-inputs">
+                                    <input type="text" class="form-control" name="group_number" required placeholder="<?php echo xla('Required'); ?>">
+                                </div>
+                            </div>
+                            <div class="form-group inline">
+                                <label class="control-label" for=""><?php echo xlt('Policy Begin Date')?></label>
+                                <div class="controls inline-inputs">
+                                    <input type="text" class="form-control datepicker" name="date" placeholder="<?php echo xla('Policy effective date'); ?>">
+                                </div>
+                            </div>
+                            <div class="form-group inline">
+                                <label class="control-label" for=""><?php echo xla('Co-Payment')?></label>
+                                <div class="controls inline-inputs">
+                                    <input type="number" class="form-control" name="copay" placeholder="<?php echo xla('Plan copay if known'); ?>">
+                                </div>
+                            </div>
+                        </div>
+                        <button class="btn btn-primary prevBtn btn-sm pull-left" type="button"><?php echo xlt('Previous') ?></button>
+                        <button class="btn btn-primary nextBtn btn-sm pull-right" type="button"><?php echo xlt('Next') ?></button>
+                    </fieldset>
+                </div>
+            </div>
+        </form>
+        <!-- End Insurance. Next what we've been striving towards..the end-->
+        <div class="row setup-content" id="step-4" style="display: none">
+            <div class="col-xs-6 col-md-offset-3 text-center">
+                <div class="col-md-12">
+                    <fieldset>
+                        <legend class='bg-success'><?php echo xlt('Register') ?></legend>
+                        <div class="well" style="text-align: center">
+                            <h4 class='bg-success'><?php echo xlt("All set. Click Send Request below to finish registration") ?></h4>
+                            <hr>
+                            <p>
+                            <?php echo xlt("An e-mail with your new account credentials will be sent to the e-mail address supplied earlier.
+                            You may still review or edit any part of your information by using the top step buttons to go to the appropriate panels.
+                            Note to be sure you have given your correct e-mail address.
+                            If after receiving credentials and you have trouble with access to the portal, please contact administration.")?>
```

Comment:
Do you need the end lines in this display?
If not, then should make this 1 ugly line to allow translations.(new lines can not be in translation constants)